### DISCUSSION TITLE
Open Suggestion popup programmatically

### DISCUSSION THREAD

**Discussion by dMcCaffree** [ueberdosis/tiptap#5618] [General Questions]
I can't seem to find anywhere in the documentation (or code) that would allow for opening the Suggestion popup programmatically.

I'm wondering if I'm missing something or if this simply isn't possible yet.

I was looking for something like `editor.commands.suggestions.open()`

Thank you so much!

**Comment by DrewJohnsonGT** [comment]
I was trying to open a suggestion popup from another suggestion popup's menu items.
Ended up just using the other suggestion menus trigger character to open the menu programmatically
i.e. in the command for the first suggestion extension's menu - 
```
  command: ({ editor, range }: { editor: Editor; range: Range }) => {
    editor
      .chain()
      .focus()
      .deleteRange(range)
      .insertContent(TRIGGER_CHAR)
      .run();
  },
```
where `TRIGGER_CHAR` is the trigger for the menu you want to open next
